src/app/GitUI/CommandsDialogs/FormRemotes.cs

```diff
@@ -61,10 +61,10 @@ static SortableGitRefList()
             new("Select ssh key file");
 
         private readonly TranslationString _labelUrlAsFetch =
-            new("Fetch Url");
+            new("Fetch &Url");
```

Comment:
Is it a good idea to give the same access key for 2 different  labels? What is the behavior ? Shouldn't we use 'f' instead ?

Reply:
Ideally it should be a single mnemonic per control. Ideally, the same operations across the app should use the same mnemonics (e.g., if one dialog uses "&Fetch" and another uses "F&etch" - it is a bad UX).
Also, strictly speaking, not all controls require a mnemonic (as an example look though VS menus; in the MS Office suite not every has mnemonics either, though it's harder to see with the ribbons).